lib/screens/lobby.dart

```diff
@@ -20,14 +24,108 @@ class LobbyScreen extends StatelessWidget {
               };
 
               context.read<NavigationService>().showConfirmationDialog(
-                  onConfirmed: signOut,
-                  confirmText: 'Sign Out',
-                  cancelText: 'Return');
+                    onConfirmed: signOut,
+                    confirmText: 'Sign Out',
+                    cancelText: 'Return',
+                  );
             },
             child: Text('log out'),
           )
         ],
       ),
+      body: Container(
+        child: StreamBuilder<List<Game>>(
+            stream: context.watch<DatabaseService>().streamOfCreatedGames,
+            builder: (context, snapshot) {
+              if (snapshot.data == null) {
+                print('still null');
+                return Container();
+              } else {
+                print(' stream builder working');
+                return Padding(
+                  padding: const EdgeInsets.fromLTRB(0, 10, 0, 0),
+                  child: ListView(
+                    children: <Widget>[
+                      for (var game in snapshot.data) GameTile(game)
+                    ],
+                  ),
+                );
+              }
+            }),
+      ),
+    );
+  }
+}
+
+class GameTile extends StatelessWidget {
```

Comment:
awesome that you pulled this out into a separate widget!  what do you think about putting it in its own file?